### ISSUE TITLE
slugify option 'latin' unreleased

### ISSUE THREAD

**Issue by festwertspeicher** [jekyll/jekyll#6549]
- [x] I updated to the latest Jekyll (or) if on GitHub Pages to the latest `github-pages`
- [x] I am on (or have tested on) ***Windows*** 10+

## My Reproduction Steps

I just tested this bug locally and not on GitHub Pages. I'm on Windows 10 and using Bash on Ubuntu on Windows. The slugify option 'latin' is not doing anything at all. This test-code 

```
<p>{{ "The _cönfig.yml file" | slugify }}</p>
<p>{{ "The _cönfig.yml file" | slugify: 'pretty' }}</p>
<p>{{ "The _cönfig.yml file" | slugify: 'latin' }}</p>
```

renders

> the-cönfig-yml-file
> the-_cönfig.yml-file
> the _cönfig.yml file

## The Output I Wanted

> the-cönfig-yml-file
> the-_cönfig.yml-file
> the-config-yml-file

as mentioned in the [documentation](https://jekyllrb.com/docs/templates/#options-for-the-slugify-filter)

**Comment by DirtyF** [comment]
@festwertspeicher The docs are always a bit early, #6509 has recently been merged but not been released yet.

You can only test this feature if you [tell Bundler to use this repo as the gem](https://bundler.io/v1.16/guides/git.html#gems-from-git-repositories)

**Comment by parkr** [comment]
@DirtyF does the documentation need to be updated? Or is this good to close as answered?

**Comment by DirtyF** [comment]
Should we add a version badge for newest features ?

![version-badge](https://user-images.githubusercontent.com/103008/32771735-88fa214e-c8d8-11e7-8711-fdf3949ff4bd.png)

**Comment by ashmaroli** [comment]
Awesome idea!! :+1:  :+1:

**Comment by ashmaroli** [comment]
@DirtyF what happened to the above idea?

**Comment by DirtyF** [comment]
@ashmaroli I didn't see everyone was on board, just added an issue to discuss implementation details (where and when should we add this badge).

**Comment by ashmaroli** [comment]
Addressed via #6574
